kstreams/prometheus/monitor.py

```diff
@@ -124,6 +125,19 @@ def _clean_consumer_metrics(self) -> None:
         self.MET_POSITION.clear()
         self.MET_HIGHWATER.clear()
 
+    async def _clean_stream_consumer_metrics(self, stream: Stream) -> None:
```

Comment:
The `async` is not needed. Also, I think you can make this method public (remove the `_`)